website/screens/components/button/code/ButtonCodePage.tsx

```diff
@@ -63,60 +73,89 @@ const sections = [
               button label.
             </td>
+            <td>-</td>
           </tr>
           <tr>
-            <td>iconPosition: 'before' | 'after'</td>
+            <td>iconPosition</td>
             <td>
-              <Code>'before'</Code>
+              <TableCode>'before' | 'after'</TableCode>
             </td>
             <td>Whether the icon should appear after or before the label.</td>
+            <td>
+              <TableCode>'before'</TableCode>
+            </td>
           </tr>
           <tr>
-            <td>disabled: boolean</td>
+            <td>disabled</td>
             <td>
-              <Code>false</Code>
+              <TableCode>boolean</TableCode>
             </td>
             <td>If true, the component will be disabled.</td>
+            <td>
+              <TableCode>false</TableCode>
+            </td>
           </tr>
           <tr>
-            <td>onClick: function</td>
-            <td></td>
+            <td>onClick</td>
+            <td>
+              <TableCode>{"() => void"}</TableCode>
+            </td>
             <td>
               This function will be called when the user clicks the button.
             </td>
+            <td>-</td>
           </tr>
           <tr>
-            <td>margin: string | object</td>
-            <td></td>
+            <td>margin</td>
             <td>
-              Size of the margin to be applied to the component ('xxsmall' |
-              'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge').
-              You can pass an object with 'top', 'bottom', 'left' and 'right'
-              properties in order to specify different margin sizes.
+              <TableCode>
+                'xxsmall' | 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' |
+                'xxlarge' | Margin
+              </TableCode>
             </td>
+            <td>
+              Size of the margin to be applied to the component. You can pass an
+              object with 'top', 'bottom', 'left' and 'right' properties in
+              order to specify different margin sizes.
+            </td>
+            <td>-</td>
           </tr>
           <tr>
-            <td>size: string </td>
+            <td>size</td>
             <td>
-              <Code>'fitContent'</Code>
+              <TableCode>
+                'small' | 'medium' | 'large' | 'fillParent' | 'fitContent'
+              </TableCode>
             </td>
+            <td>Size of the component.</td>
             <td>
-              Size of the component ('small' | 'medium' | 'large' | 'fillParent'
-              | 'fitContent').
+              <TableCode>'fitContent'</TableCode>
             </td>
           </tr>
           <tr>
-            <td>tabIndex: number</td>
+            <td>tabIndex</td>
+            <td>
+              <TableCode>number</TableCode>
+            </td>
+            <td>
+              Value of the <Code>tabindex</Code> attribute.
+            </td>
```

Comment:
Please change `tabindex` to `tabIndex`.

Reply:
`tabindex` is the HTML attribute. I wrote it like that so the user can associate the React prop with its HTML counterpart.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Sure, I realised now reviewing the Checkbox!